src/EnergyPlus/SingleDuct.cc

```diff
@@ -4493,6 +4494,25 @@ namespace SingleDuct {
             FanOp = 0;
             CalcVAVVS(SysNum, FirstHVACIteration, ZoneNodeNum, HCType, 0.0, 0.0, FanType, MassFlow, FanOp, QDelivered);
         }
+
+        //MassFlow = Node(SysInletNode).MassFlowRate;
+        // Move data to the damper outlet node
+        SysOutlet(SysNum).AirTemp = SysInlet(SysNum).AirTemp;
+        SysOutlet(SysNum).AirHumRat = SysInlet(SysNum).AirHumRat;
+        SysOutlet(SysNum).AirMassFlowRate = MassFlow;
+        SysOutlet(SysNum).AirMassFlowRateMaxAvail = SysInlet(SysNum).AirMassFlowRateMaxAvail;
+        SysOutlet(SysNum).AirMassFlowRateMinAvail = SysInlet(SysNum).AirMassFlowRateMinAvail;
+        SysOutlet(SysNum).AirEnthalpy = SysInlet(SysNum).AirEnthalpy;
+
+        // calculate damper Position.
+        if (Sys(SysNum).AirMassFlowRateMax == 0.0) {
+            Sys(SysNum).DamperPosition = 0.0;
+        } else {
+            Sys(SysNum).DamperPosition = MassFlow / Sys(SysNum).AirMassFlowRateMax;
+        }
```

Comment:
calculate the vav damper position for reporting purpose

Reply:
:+1: I see the unit test below exercises this calculation.